plugins.txt

```diff
@@ -41,6 +41,10 @@ mailer  # Mailer
 
 ansicolor  # adds support for ANSI escape sequences, including color, to Console Output.
 configuration-as-code  # allows fully bootstrapped Jenkins without any manual configurations
-configuration-as-code-support
 job-dsl  # allows programmatic creation of projects using a DSL
 envinject  # Environment Injector: setup a custom environment for jobs
+bouncycastle-api:2.18  # update required by ghprb plugin
```

Comment:
Please unpin all version.  I did not pin all the other plugin versions on purpose so update it's easier to update them all, see README.md

===
We do not pin exact versions and dependencies of all the plugins so it's faster to get the latest on each rebuild of the docker image without having to edit any hardcoded committed versions before a rebuild.

Reproducibility is ensured by the docker image. Something do not work, just rollback to the previous docker image. Need to find out what was the previous version of a plugin, look in the previous docker image.

We can still pin a specific version in plugins.txt if we need to. The syntax is plugin-name:version, ex: pipeline-build-step:2.7.

===

Reply:
I've seen the message before committing and yes, it makes sense for most plugins.
However, I would prefer to keep the pin for "ghprb" since on version 1.40 there was a security issue (plain text credentials stored on disk), that was fixed in later version. 
So that way if a similar issue occurs, we wouldn't be impacted before upgrading the plugin.